### DISCUSSION TITLE
Bulk Add IP Addresses

### DISCUSSION THREAD

**Discussion by marcossiqueira** [netbox-community/netbox#6803] [General]
Hello everyone

I have a problem registering IPV6 networks.
I'm mass adding IPV6 addresses to a /48 block, but the application is failing to run.
Can anybody help me?

I'm using this formula
2800:1e0:1010::[1-ffff]/128

**Comment by candlerb** [comment]
That's adding 65,535 separate objects, and I suspect is hitting a timeout - and then the database transaction is rolled back.  Does it work with smaller blocks, e.g. `2800:1e0:1010::[1-1f]/128` ?  For me that works (with Netbox v2.11.9) but takes about 5 seconds.

It is possible to increase your request timeout in [gunicorn](https://github.com/netbox-community/netbox/blob/develop/contrib/gunicorn.py#L11-L12) and in your frontend proxy (apache or nginx).  Alternatively you can add the addresses in smaller batches, or to use a CLI script or a program which talks to the REST API rather than going through the web UI.

**Reply by sdktr** [reply]
V3 has IP range as a dedicated type, maybe this suits your use case
